app/views/users/team.html.haml

```diff
@@ -5,7 +5,7 @@
     .column-two-thirds
       %h1.heading-large= t('views.users.team.page_title')
     .column-one-third
-      = link_to t('views.users.new.page_title'), new_user_invitation_path, class: "button align-with-heading"
+      = link_to t('views.users.new.page_title'), new_user_invitation_path('advanced'), class: "button align-with-heading"
```

Comment:
this is incorrect but I think its again for another reason. I think you injected this as a way to give custom layouts for different types of user but now that we use cancancan we can probably remove the injection altogether and just use `new_user_invitation_path`